get_together/templates/get_together/events/show_event.html

```diff
@@ -202,6 +202,19 @@ <h2>{{ event.name }}</h2>
             </div>
             {% endif %}
 
+            <div class="row mb-2">
+              <div class="col-12">
+                <iframe
+                  width="100%"
+                  height="300"
+                  frameborder="0" style="border:0"
+                  src="https://www.google.com/maps/embed/v1/place?key={{ settings.GOOGLE_MAPS_API_KEY }}&q={{event.place.address|urlencode}}%2C%20{{event.place.city|urlencode}}&zoom=14"
```

Comment:
So, we are using the place address and city to filter the place on google maps. I'm not sure if this is a good approach. Currently, the places can be inputted by the user, and there's no guarantee the entered address is correct and can be found in google maps.

@pawalls , any idea here? In the future, It'll be interesting to implement a smooth integration with google maps. Do you think we can use the address for now?

Reply:
Originally the app used the name of the place and city to filter the place.  The upside is the place's name can show in the map, if the place on Google is exactly the one the user is looking for. Sometimes there are discrepancies, such as when the place is not in Google's Place library. That's why I chose to use the address and city. I'd love to hear your suggestions.

Reply:
@thiagoferreiraw -- would your concerns be alleviated if we used Google Map search for selecting location during event creation? I believe we'd end up with a lat/long or place id in the database and then would use the maps API to translate that back to a place.

Reply:
yes @pawalls! Seems much less likely to fail. Can we log an issue and add to the board for this?

Reply:
@thiagoferreiraw -- done.